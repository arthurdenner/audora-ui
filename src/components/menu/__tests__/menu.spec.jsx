import React from 'react';
import { render, mount } from 'enzyme';
import Menu from '../menu';

describe('Menu', () => {
  it('renders correctly', () => {
    const wrapper = render(<Menu
      items={[
        {
          key: 1,
          name: 'item1',
          content: 'Item 1',
          onClick: (e, data) => console.log(e, data),
        }]}
    />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly when minimal', () => {
    const wrapper = mount(<Menu
      items={[
        {
          key: 1,
          name: 'item1',
          content: 'Item 1',
          onClick: (e, data) => console.log(e, data),
        }]}
      minimal
    />);
    expect(wrapper.props().minimal).toBe(true);
  });
});
