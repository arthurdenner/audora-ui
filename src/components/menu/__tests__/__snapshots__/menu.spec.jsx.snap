// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<div
  class="sc-EHOje ebXamK"
>
  <div
    class="sc-bZQynM bhfddE"
  >
    <button
      class="sc-gZMcBi bQbqXU sc-bxivhb giYbNp"
      title="Item 1"
      type="button"
    >
      Item 1
    </button>
  </div>
</div>
`;
